tests/tests_modifications_with_selinux_disabled.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: Ensure the default is targeted, enforcing, without local modifications
+  hosts: all
+  gather_facts: true
+  vars:
+    selinux_all_purge: true
+    selinux_logins_purge: true
+
+  tasks:
+    - name: Ensure selinux-policy-targeted
+      package:
+        name:
+          - selinux-policy-targeted
+        state: present
+
+    - name: Add a Linux System Roles SELinux User
+      user:
+        comment: Linux System Roles SELinux User
+        name: sar-user
+    - name: Switch to permissive to allow login when selinuxfs is not mounted
+      command: setenforce 0
+      changed_when: true
+      when: ansible_selinux.status != "disabled" and
+        ansible_selinux.status != "permissive"
+    - name: Get selinuxfs mountpoint
+      command: findmnt -n -t selinuxfs --output=target
+      changed_when: false
+      register: selinux_mountpoint
+    - name: >-
+        Umount selinux mountpoint to emulate SELinux disabled
+        system {{ selinux_mountpoint.stdout }}
+      command: umount -l {{ selinux_mountpoint.stdout }}
+      changed_when: true
+      when: selinux_mountpoint.stdout != ""
+
+    - name: Execute the role and catch errors
+      block:
+        - name: Include role
+          include_role:
+            name: linux-system-roles.selinux
+          vars:
+            # https://github.com/ansible-collections/ansible.posix/pull/496
+            # selinux_booleans:
+            #  - {name: 'ssh_sysadm_login', state: 'off', persistent: 'yes'}
+            selinux_fcontexts:
+              - {target: '/tmp/test_dir(/.*)?', setype: 'user_home_dir_t',
+                 ftype: 'd'}
+            selinux_ports:
+              - {ports: '22100', proto: 'tcp', setype: 'ssh_port_t',
+                 state: 'present'}
+            selinux_logins:
+              - {login: 'sar-user', seuser: 'staff_u',
+                 serange: 's0-s0:c0.c1023', state: 'present'}
```

Comment:
did you mean to have a `rescue` block here to catch errors?

Reply:
No, error would mean that the test failed.

The `rescue` used in https://github.com/linux-system-roles/selinux/blob/main/tests/tests_selinux_disabled.yml#L73 is necessary as the role task fails/errors when reboot is required but it hides another errors. And this test should expose them.

Reply:
And the role settings in this test does not require reboot.

Reply:
> OTOH, it should probably do a cleanup in `rescue`. Is there a way to report an error but continue with the cleanup steps?

yes - block/rescue/always

```yaml
block:
  taskA
  taskB # may throw an error
  taskC # might not reach here
rescue:
  handle error - check if error is the expected one and not something else
always:
  cleanup taskC if needed
  cleanup taskB if needed
  cleanup taskA
```